torchaudio/backend/common.py

```diff
@@ -149,6 +179,10 @@ def __init__(self,
 
 _INFO_DOCSTRING = r"""Gets metadata from an audio file without loading the signal.
 
+Warnings:
```

Comment:
I think this is unnecessary, because it's duplicated with the warning statement and because we don't do this in any other library.